partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/BlacklistedTests.txt

```diff
@@ -96,14 +95,12 @@ run/various-flat-classpath-types.scala
 run/java-erasure.scala
 run/t2250.scala
 
-# Uses java.math.BigDecimal / BigInteger
-run/bigDecimalCache.scala
+# Uses java.math.BigDecimal / BigInteger : but failures not due to them
 run/hashhash.scala
 run/is-valid-num.scala
-run/range.scala
 run/stringinterpolation_macro-run.scala
 run/t5356.scala
-run/t6064.scala
+
```

Comment:
You also need to adapt the test lists in the other directories. (not only 2.11.6)